sw/host/hsmtool/src/commands/kdf/mod.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+use cryptoki::session::Session;
+use serde::{Deserialize, Serialize};
+use std::any::Any;
+
+use crate::commands::Dispatch;
+use crate::module::Module;
+
+pub mod export;
+pub mod import;
+
+#[derive(clap::Subcommand, Debug, Serialize, Deserialize)]
+pub enum Kdf {
+    Export(export::Export),
+    Import(import::Import),
+}
```

Comment:
Are there other operations we'll add in the future to `kdf` commands? (e.g. keygen and/or hmac)

Reply:
I think that it would be useful to add `generate` and `derive` commands. I will follow up with changes in a future PR as part as we make progress on enabling the reference provisioning flows.